DS13/code/modules/borg/antag_datum.dm

```diff
@@ -0,0 +1,119 @@
+#define isBorgDrone(D) (D in GLOB.borg_collective.drones)
+
+/datum/antagonist/borg_drone
+	name = "Borg Drone"
+	antagpanel_category = "Borg Drone"
+	job_rank = ROLE_BORG_DRONE
+	show_name_in_check_antagonists = TRUE
+	antag_moodlet = /datum/mood_event/focused
+
+/datum/antagonist/borg_drone/on_gain()
+	. = ..()
+	antag_memory += "We are a <font color='red'><B>borg drone</B></font>. We must assimilate the station at all costs."
+	if(!isBorgDrone(src))
+		GLOB.borg_collective.drones += owner.current //They're in the collective, but need the conversion table for all their upgrades like the tool etc.
+		GLOB.borg_collective.check_completion()
+	var/datum/objective/O = new /datum/objective/survive() //Change this later
+	O.owner = owner
+	objectives += O
+	borgify(owner.current)
+
+/datum/antagonist/borg_drone/on_removal()
+	to_chat(owner.current,  "We are no longer a <font color='red'><B>borg drone</B></font>")
+	unborgify(owner.current)
+	GLOB.borg_collective.drones -= owner.current //They're in the collective, but need the conversion table for all their upgrades like the tool etc.
+	. = ..()
+
+/datum/antagonist/borg_drone/greet()
+	SEND_SOUND(owner.current, 'DS13/sound/effects/borg/collectivewhisper.ogg')
+	to_chat(owner.current, "<span_class='bigbold'>We are borg.</span>")
+	to_chat(owner.current, "<span_class='danger'>We require additional parts. We must seek out an assimilation bench to improve ourselves.</span>")
+	owner.announce_objectives()
+
+/datum/antagonist/borg_drone/proc/borgify(mob/living/carbon/human/H = owner.current)
+	GLOB.borg_collective.drone_count ++
+	H.skin_tone = "albino"
+	H.eye_color = "red"
+	H.underwear = "Nude"
+	H.undershirt = "Nude"
+	H.socks = "Nude"
+	H.hair_style = "Bald"
+	H.hygiene = 100000000 //Smelly borg BEGONE
+	H.add_trait(TRAIT_CLUMSY, "borg")
```

Comment:
Clumsy?

Reply:
So they don’t use guns and cuffs I suppose

Reply:
As Alex says